services/hanke-service/src/main/resources/db/changelog/changesets/018-add-haittojen-hallinta-fields.yml

```diff
@@ -0,0 +1,35 @@
+databaseChangeLog:
+  - changeSet:
+      id: 018-add-haittojen-hallinta-fields
+      comment: Add fields for managing haittojen hallinta
+      author: Niko Junkala
+      changes:
+        - addColumn:
+            tableName: hanke
+            columns:
+                - column: { name: pyoraliikenteenpaareitit, type: text }
```

Comment:
Harkitsiko muita tapoja tallentaa näitä tietoja kuin 26 uutta tekstisaraketta? Niinkuin joku JSON-rakenne tai erillinen taulu, johon tallennetaan avain-arvo pareja?